bin/cli.sh

```diff
@@ -5,16 +5,14 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/
 #
 set -e
-script_dir="$(dirname "${BASH_SOURCE[0]}")"
-if [[ "$script_dir" == *".bin" ]] ; then
-    root_dir="$(cd "$script_dir/../uniffi-bindgen-react-native" && pwd)"
-elif [[ "$script_dir" == *"bin" ]] ; then
-    root_dir="$(cd "$script_dir/.." && pwd)"
-else
-    echo "Unable to locate the uniffi-bindgen-react-native directory" 2>/dev/null
-    exit 1
-fi
 
+# Use Node to resolve the path to the module. The require.resolve
+# function returns a thing like:
+#
+# PATH_TO/node_modules/uniffi-bindgen-react-native/dist/objects.js
+#
+# So we drop the /dist and everything after it.
+root_dir=$(node --print "require.resolve('uniffi-bindgen-react-native/objects')" | sed 's/\/dist.*//')
```

Comment:
I think this should just be uniffi-bindgen-react-naive and it needs to strip out `typescript/src/index\\.ts` instead of dist.